_includes/howitworks-foragencies.html

```diff
@@ -0,0 +1,49 @@
+<div class="row">
+  <div class="icon-card col-md-3" about="" typeof="">
+    <div class="icon-card__icon">
+      <img typeof="foaf:Image" src="/assets/img/tabs/contact.svg" width="100" height="96" alt="Contractors Join" />
+    </div>
+    <div class="icon-card__title">
+      <h4>Get in touch</h4>
+    </div>
+    <div class="icon-card__description px-xl-5">
+      <p>Get in touch with  WP Contractors to let us know what you’re looking for.</p>
```

Comment:
Since this is going in the theme repository, `WP Contractors` needs to be replaced with the `{{ site.title }}` variable-

https://github.com/WPContractors/wpcontractors.github.io/blob/develop/_config.yml#L1

Reply:
I would try to put the variable but locally it only shows me the value = "Contractors". But now I understand it comes from wpcontractors.github.io with the correct value. Thanks.

Reply:
I changed...